CHANGELOG.md

```diff
@@ -20,6 +22,7 @@ Anticipated release: April 6, 2020
 
 See our [release history](https://github.com/18F/cms-hitech-apd/releases)
 
+[2112]: https://github.com/18F/cms-hitech-apd/issues/2112
```

Comment:
Don't forget to add the octothorpe!

Reply:
![eagle eyes](https://media.giphy.com/media/RKw94AILIkuWY/giphy.gif)